network/src/validator_network/test.rs

```diff
@@ -162,12 +166,14 @@ fn test_mempool_sync() {
         .spawn(network_provider.start().unit_error().compat());
 
     // The dialer dials the listener and sends a mempool sync message
-    let mut mempool_msg = MempoolSyncMsg::new();
-    mempool_msg.set_peer_id(dialer_peer_id.into());
+    let mut mempool_msg = MempoolSyncMsg::default();
+    mempool_msg.peer_id = dialer_peer_id.into();
     let sender = AccountAddress::new([0; ADDRESS_LENGTH]);
     let keypair = compat::generate_keypair(&mut rng);
-    let txn = get_test_signed_txn(sender, 0, keypair.0, keypair.1, None).into_proto();
-    mempool_msg.set_transactions(::protobuf::RepeatedField::from_vec(vec![txn.clone()]));
+    let txn: SignedTransaction = get_test_signed_txn(sender, 0, keypair.0, keypair.1, None)
+        .try_into()
```

Comment:
Getting rid of `::protobuf::RepeatedField::from_vec` is so much better.

Reply:
mhmm!